docs/auth/microsoft/provider.md

```diff
@@ -28,6 +28,7 @@ On the **API permissions** tab, click on `Add Permission`, then add the followin
 - `openid`
 - `profile`
 - `User.Read`
+- Optional custom permissions you defined in the configuration file
```

Comment:
Not sure how to understand this line. The docs say "add the following [...] permission" and this is not really specific. Could this be moved out of the listening just bellow it? Do you mean the `app-config.yaml` with the configuration files? Are the "permissions" equal to the "additionalScopes"?

Reply:
Changed the description of additional scopes.